### PR TITLE
Translated the file debugging-getting-started.md to Arabic 

### PR DESCRIPTION
**Pull request by celyes**
translated the file `debugging-getting-started.md` located in `docs/guides/` directory to Arabic.

Thank you, everyone!

@SoftVenue @Aissaoui-Ahmed @Maledong @fhemberger

### CODE REVIEW

File: locale/ar/docs/guides/debugging-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+---
+title: Debugging - Getting Started
```
Comment (Aissaoui-Ahmed) [inline locale/ar/docs/guides/debugging-getting-started.md]: translate title

**Review by Aissaoui-Ahmed — CHANGES_REQUESTED** [review]

File: locale/ar/docs/guides/debugging-getting-started.md (hunk 1 above)
Comment (celyes) [reply locale/ar/docs/guides/debugging-getting-started.md]: @Aissaoui-Ahmed Roger that

Comment (celyes) [reply locale/ar/docs/guides/debugging-getting-started.md]: @Aissaoui-Ahmed requested changes has been made to the file... take a look

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Comment by ghost** [comment]
@celyes：The build is something wrong, please re-submit again to see whether there're any other mistakes or not ;)

**Pull request merged**